web/pdf_rendering_queue.js

```diff
@@ -33,6 +33,9 @@ class PDFRenderingQueue {
     this.pdfThumbnailViewer = null;
     this.onIdle = null;
     this.highestPriorityPage = null;
+    /**
+     * @type {number}
+     */
```

Comment:
Sorry, but why?

This property really *should* be completely internal.

Reply:
because of the `setTimeout` call, `tsc` infers this type to be `idleTimeout: NodeJS.Timeout | null;`, resulting in the following compilation error since the types test project isn't building for node:
```
Couldn't compile TypeScript test: build/typestest/types/web/pdf_rendering_queue.d.ts(9,18): error TS2503: Cannot find namespace 'NodeJS'.
```

https://github.com/mozilla/pdf.js/blob/56e7bb626c89282f207b585ebed74c767c68db95/web/pdf_rendering_queue.js#L95-L97

Writing this has made me realize that pdf.js seems to support running server-side and since `NodeJS.setTimeout` returns an object while` window.setTimeout` returns a number, it would be more proper to type this as `ReturnType<typeof setTimeout>`, where `setTimeout`'s type will ultimately depend on your project setup.